assets/src/containers/Course.js

```diff
@@ -74,7 +74,7 @@ function Course (props) {
               render={props =>
                 <AssignmentPlanningV2
                   {...props}
-                  disabled={!courseInfo.course_view_options.ap}
+                  disabled={!courseInfo.course_view_options.apv2}
```

Comment:
For students they will only see V2 for winter so for then it doesn't make sense to V2 in the text. One way to handle hoping v2 is going to stay. Make the old assignment as `apv1` and new assignment as just `ap`. Making this change with `model.py` as well.